shell/components/__tests__/AsyncButton.test.ts

```diff
@@ -1,15 +1,13 @@
 import { mount, Wrapper } from '@vue/test-utils';
 import AsyncButton, { ASYNC_BUTTON_STATES } from '@shell/components/AsyncButton.vue';
-import { cleanHtmlDirective } from '@shell/plugins/clean-html-directive';
 
 describe('component: AsyncButton', () => {
   it('should render appropriately with default config', () => {
     const mockExists = jest.fn().mockReturnValue(true);
     const mockT = jest.fn().mockReturnValue('some-string');
 
     const wrapper: Wrapper<InstanceType<typeof AsyncButton> & { [key: string]: any }> = mount(AsyncButton, {
-      directives: { cleanHtmlDirective },
-      mocks:      {
+      mocks: {
```

Comment:
It was always easy to miss adding the clean html directive when writing tests, and would flood the terminal with warning messages when we did so. I'm glad to see that this can be handled in `jest.setup` now 😁